src/TimePicker/TimePickerDialog.js

```diff
@@ -86,6 +86,10 @@ class TimePickerDialog extends React.Component {
       root: {
         fontSize: 14,
         color: this.getTheme().clockColor,
+        WebkitUserSelect: 'none',
+        KhtmlUserSelect: 'none',
+        MozUserSelect: 'none',
```

Comment:
@oliviertassinari : I checked. the autoprefixer works. i made the necessary changes. Thanks for the heads up!